src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -1020,10 +1028,17 @@ MulticopterAttitudeControl::control_attitude_rates(float dt)
 
 	/* angular rates error */
 	math::Vector<3> rates_err = _rates_sp - rates;
+	math::Vector<3> derivative;
+	_foaw_rollrate.set_sample_time(dt);
+	_foaw_pitchrate.set_sample_time(dt);
+	_foaw_yawrate.set_sample_time(dt);
+	derivative(0) = _foaw_rollrate.apply(rates(0));
+	derivative(1) = _foaw_pitchrate.apply(rates(1));
+	derivative(2) = _foaw_yawrate.apply(rates(2));
 
 	_att_control = rates_p_scaled.emult(rates_err) +
-		       _rates_int +
-		       rates_d_scaled.emult(_rates_prev - rates) / dt +
+		       _rates_int -
```

Comment:
Before, we computed negative derivative and added it. Now, since the differentiator computes the derivative with the correct sign, I decided to change the sign here.